includes/Runtime.php

```diff
@@ -39,23 +39,25 @@ public function loadIntoPage( $page ) {
 	public function prepareRuntime() {
 		global $wp_version;
 		$sdk = apply_filters( 'newfold-runtime', array( 'wpversion' => $wp_version ) );
-		return array(
-			'site'         => array(
-				'url'   => \get_site_url(),
-				'title' => htmlspecialchars_decode( \get_bloginfo( 'name' ) ),
-			),
-			'admin_url'    => \admin_url(),
-			'adminUrl'		=> \admin_url(),
-			'base_url'     => \get_home_url() . '/index.php',
-			'homeUrl'     => \get_home_url(),
-			'capabilities' => $this->container->get( 'capabilities' )->all(),
-			'sdk'          => $sdk, // kept for backward compatability, will be removed later
-			'siteUrl'		=> \get_site_url(),
-			'siteTitle' => htmlspecialchars_decode( \get_bloginfo( 'name' ) ),
-			'restUrl' 	=> \esc_url_raw( \get_home_url() . '/index.php?rest_route=' ),
-			'restNonce'			=> wp_create_nonce('wp-rest'),
-			'isWoocommerceActive' 	=> is_plugin_active('woocommerce/woocommerce.php'),
-			...$sdk
+		return array_merge( 
```

Comment:
@circlecube We should dispose of the array merge part. There's no need to duplicate the SDK contents into the root. We should manually add the `wpversion` attribute to the root (but name it `wpVersion` instead).